docker-compose-dev.yml

```diff
@@ -7,19 +7,30 @@ services:
     build:
       context: ./
       dockerfile: Dockerfile.dev
-    # Not sure why ports aren't working. Using host as a hack
-    # ports:
-    #   - "9999:9999"
-    network_mode: "host"
+    ports:
+      - "9999:9999"
```

Comment:
Awesome that you figured this out! It really bugged me that it didn't work. Especially since I've done it a number of times before with no issue